tests/lang/basics.pk

```diff
@@ -36,4 +36,15 @@ m = {}
 m['m'] = m
 assert(to_string(m) == '{"m":{...}}')
 
+# Bitwise operation tests
+assert((1 | 1) == 1)
+assert((1 | 4) == 5)
+assert((3 | 5) == 7)
+
+assert((1 & 1) == 1)
+assert((1 & 2) == 0)
+assert((4 & 7) == 4)
+
+# If we got here with no errors, then all test passed
+print('All tests PASSED ✅')
```

Comment:
pocketlang currently doesn't support utf8 strings yet ([in TODO](https://github.com/ThakeeNathees/pocketlang/blob/473daa4716bcccf58add33d8f952665a90bf9411/docs/TODO.txt#L30)), don't use them for now. also for `core.pk`